GenderPayGap.WebUI/Views/Viewing/EmployerDetails/Employer.cshtml

```diff
@@ -101,18 +101,27 @@
                             {
                                 <tr>
                                     <td colspan="2" style="border-bottom: none;">
-                                        @if (organisation.GetIsInscope(report.AccountingDate.Year))
+                                        @if (Global.ReportingStartYearsToExcludeFromLateFlagEnforcement.Contains(report.AccountingDate.Year) 

```

Comment:
This logic means that
- we will only see this message if we are excluding this year
- and if the excluded year is 2019

So if we do turn enforcement back on, then this message will not be shown